README.md

```diff
@@ -230,8 +230,8 @@ Keys supported by image output:
 * `unpack=true`: unpack image after creation (for use with containerd)
 * `dangling-name-prefix=[value]`: name image with `prefix@<digest>` , used for anonymous images
 * `name-canonical=true`: add additional canonical name `name@<digest>`
-* `compression=[uncompressed,gzip]`: choose compression type for layer, gzip is default value
-
+* `compression=[uncompressed,gzip]`: choose compression type for layers newly created and cached, gzip is default value
+* `force-exporting-compression=[uncompressed,gzip]`: choose compression type for all layers (including already existing layers) for exporting. compression type specified by `compression` flag is respected by default.
```

Comment:
Option name is too long. I suggest `compression-force` or `compression-all`.

Reply:
Or can we replace the old `compression` opt? Was it ever useful?

Reply:
Renamed to `compression-all`.